### DISCUSSION TITLE
Worth Getting Latest Unstable Version?

### DISCUSSION THREAD

**Discussion by NazmusLabs** [otya128/winevdm#1046] [General]
Hello, so I noticed that the last "stable" (or precompiled) release was over two years ago! I seemed to be having minor issues with one game (Chip's Challenge for Windows 3.1) on Windows 11.

So I am wondering that has there been enough significant changes in the last two years for it to be worthwhile for me to compile the latest code and use that? I.E., are there enough improvements and/feature changes to forge a bit of stability of the now 2+ year old build?

Thanks, and May Allah (S.W.T.) bestow upon you His Blessings and Guidance.

**Comment by Anixx** [comment]
There is no need to compile, you can download it in ready-to-use form.
